README.md

```diff
@@ -35,7 +37,8 @@ State-of-the-art models in AI,ML,NLP & Vision for FinTech community by Yubi's Da
     * [Fintech Document Segmentation model v1](./yubiai/vision/document_segmentation/)
 * April 2023
     * [Fintech Document Segmentation model v2](./yubiai/vision/document_segmentation/)
-
+* Jun 2023
+    * [Document Image Detection](./yubiai/vision/document_image_detection/)
```

Comment:
Add entry for Image NSFW here .. even though both point to same link

Reply:
Added